core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -102,7 +103,14 @@ export class DotEmaShellComponent implements OnInit {
      */
     readonly $updateQueryParamsEffect = effect(() => {
         const params = this.uveStore.$friendlyParams();
-        this.#updateLocation(params);
+
+        const { data } = this.#activatedRoute.snapshot;
+
+        const baseClientHost = data?.uveConfig?.url;
+
+        const cleanedParams = normalizeQueryParams(params, baseClientHost);
+
+        this.#updateLocation(cleanedParams);
```

Comment:
We can do it here to reuse this code:

https://github.com/dotCMS/core/blob/7716efdc04ae6e7c45106012e0c606bbbf425914/core-web/libs/portlets/edit-ema/portlet/src/lib/store/dot-uve.store.ts#L144-L150

Nice to have, not a blocker